modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java

```diff
@@ -364,14 +364,33 @@ protected void deleteLatestVersion(StagedModel stagedModel)
 	protected void exportImportPortlet(String portletId) throws Exception {
 		exportImportPortlet(
 			portletId, new LinkedHashMap<String, String[]>(),
-			new LinkedHashMap<String, String[]>());
+			new LinkedHashMap<String, String[]>(), true, false);
```

Comment:
Don't forget to change the order in these cases, i.e, `(...,false, true)`

Reply:
Yes, sure! 🙂